README.md

```diff
@@ -0,0 +1,9 @@
+# Earthquakes
+
+Application to display Recent Earthquake data by querying [Earthquake USGS](https://earthquake.usgs.gov/earthquakes/feed/v1.0/geojson.php) using RxJava, Retrofit and GSON.
+
+![gui](/Users/leahfeiner/IdeaProjects/feiner-earthquake-2024/screenshots)
```

Comment:
This doesn't work

Reply:
This file path doesn't exist on GitHub. 

And it's pointing to a directory, not an image.